tardis/plasma/tests/conftest.py

```diff
@@ -213,12 +210,10 @@ def electron_densities(phi_saha_lte, partition_function, number_density):
 
 @pytest.fixture
 def delta(w, ionization_data, beta_rad, t_electrons, t_rad, beta_electron,
-          levels, chi_0):
-    delta_input = None
-    delta_module = RadiationFieldCorrection(None)
-    delta_module.chi_0_species = (2, 2)
+          levels):
+    delta_module = RadiationFieldCorrection(chi_0_species=(2,2))
     return delta_module.calculate(w, ionization_data, beta_rad, t_electrons,
-        t_rad, beta_electron, delta_input, chi_0)
+        t_rad, beta_electron)
 
```

Comment:
Sorry - should we sit this afternoon for 20 minutes and do the series and this one? That's probably faster than trying to answer these via github.